docs/sources/dashboards/previews.md

```diff
@@ -18,7 +19,7 @@ Dashboard previews provide an overview of all available dashboards. They help yo
 
 > **Note:** Dashboard previews are available in Grafana 9.0+ as an opt-in beta feature. Data source permissions are not yet taken into the account when displaying the dashboard previews - refer to the [permissions]({{< relref "#preview-visibility">}}) to learn more before enabling the feature.
 
-You can view dashboard previews only when the feature is enabled for your instance. It is an opt-in feature that is, by default, disabled. You can view the previews in the Grafana UI after clicking on the _Show previews_ toggle once the administrator enables the feature following the procedure described below.
+You can view dashboard previews after the administrator enables the feature following the procedure described below, and after you select the new grid layout. The dashboard previews feature is an opt-in feature that is disabled by default.
```

Comment:
```suggestion
You can view dashboard previews after an administrator activates the feature and after you select the new grid layout. The procedure is outlined below. The dashboard previews feature is an opt-in feature that is deactivated by default.
```

Reply:
after slack thread - @brendamuir wdyt about:


```
The dashboard previews feature is an opt-in feature that is disabled by default. You can view dashboard previews after an administrator enables the feature, and after you select the new grid layout. The feature-enablement procedure is outlined in the next section.
```

Reply:
in the following section. otherwise, perfect-thanks!